### PR TITLE
Nav Bar Styling

### PR DESCRIPTION
**Pull request by Drewbi**
Changes a bunch of styling to the nav bar as well as some testing on landing page

### CODE REVIEW

File: client/src/components/ProfileMenu/index.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    >
+      <MenuItem className={styles.menuitem} onClick={handleClose}>
+        <Link to="/profile">Profile</Link>
+      </MenuItem>
+      <MenuItem className={styles.menuitem} onClick={logout}>
+        Log Out
+      </MenuItem>
+    </Menu>
+  );
+  console.log(user);
```
Comment (khattakdev) [inline client/src/components/ProfileMenu/index.js]: Remove console.log

File: client/src/layout/Header/index.js [hunk 2]
```diff
@@ -19,37 +25,40 @@ const links = [
...
-        {isAuthenticated ? "Log out" : "Log in"}
-      </button>
-      {error && (
-        <div>Oops, Somethign went wrong... {JSON.stringify(error)}</div>
-      )}
-      {user && user.name}
-    </header>
+    <AppBar position="static" className={styles.header}>
+      <Toolbar>
+        <img className={styles.logo} src="/Logo.svg"></img>
```
Comment (khattakdev) [inline client/src/layout/Header/index.js]: importing `svg` in `img` tags is not a good idea
Use `<Logo className={classes.logo} />` and import it by `import { ReactComponent as Logo} from "../../...." `;

File: client/src/pages/Landing/index.js [hunk 3]
```diff
@@ -5,6 +5,17 @@ const Landing = () => {
   return (
     <>
       <h1>Landing</h1>
+      <img src="/Landing-Shared.svg"></img>
+      <img src="/github.svg"></img>
```
Comment (khattakdev) [inline client/src/pages/Landing/index.js]: import them using `ReactComponent` e.g. `import { ReactComponent as GithubLogo } from "../../"; ` and use it as a component ` <GithubLogo />`

File: client/src/pages/Landing/index.js [hunk 4]
```diff
@@ -5,6 +5,17 @@ const Landing = () => {
...
+      <svg
+        height="32"
+        viewBox="0 0 16 16"
+        version="1.1"
+        width="32"
+        aria-hidden="true"
+      >
+        <path d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27.68 0 1.36.09 2 .27 1.53-1.04 2.2-.82 2.2-.82.44 1.1.16 1.92.08 2.12.51.56.82 1.27.82 2.15 0 3.07-1.87 3.75-3.65 3.95.29.25.54.73.54 1.48 0 1.07-.01 1.93-.01 2.2 0 .21.15.46.55.38A8.013 8.013 0 0016 8c0-4.42-3.58-8-8-8z"></path>
+      </svg>
```
Comment (khattakdev) [inline client/src/pages/Landing/index.js]: Make a `.svg` for this one and import it with an appropriate name

File: client/src/pages/Profile/index.js [hunk 5]
```diff
@@ -1,183 +1,227 @@
...
-        )
-    }
+  constructor(props) {
+    super(props);
+    this.ResheduleInActive = this.ResheduleInActive.bind(this);
+    this.ResheduleActive = this.ResheduleActive.bind(this);
+    this.state = { showRegister: false };
+  }
+  ResheduleInActive() {
+    this.setState({ ...this.state, showRegister: false });
```
Comment (khattakdev) [inline client/src/pages/Profile/index.js]: Change this to 
```suggestion
    this.setState({showRegister: false });
```
React's `setState` ask only for those values you want to update

File: client/src/pages/Profile/index.js [hunk 6]
```diff
@@ -1,183 +1,227 @@
...
+  }
+  ResheduleActive() {
+    this.setState({ ...this.state, showRegister: true });
```
Comment (khattakdev) [inline client/src/pages/Profile/index.js]:
```suggestion
    this.setState({ showRegister: true });
```

File: client/src/pages/Profile/index.js [hunk 7]
```diff
@@ -1,183 +1,227 @@
...
+    this.clickMeeting = this.clickMeeting.bind(this);
+    this.state = { activeProfile: true, isProfile: true, isMeeting: false };
+  }
+  clickProfile(props) {
+    this.setState({
+      ...this.state,
+      activeProfile: true,
+      isProfile: true,
+      isMeeting: false,
+    });
```
Comment (khattakdev) [inline client/src/pages/Profile/index.js]:
```suggestion
    this.setState({
      activeProfile: true,
      isProfile: true,
      isMeeting: false,
    });
```

File: client/src/pages/Profile/index.js [hunk 8]
```diff
@@ -1,183 +1,227 @@
...
   }
+  clickMeeting(props) {
+    this.setState({
+      ...this.state,
+      activeProfile: false,
+      isProfile: false,
+      isMeeting: true,
+    });
```
Comment (khattakdev) [inline client/src/pages/Profile/index.js]:
```suggestion
    this.setState({
      activeProfile: false,
      isProfile: false,
      isMeeting: true,
    });
```

**Review by khattakdev — CHANGES_REQUESTED** [review]

File: client/src/pages/Profile/index.js (hunk 7 above)
Comment (Drewbi) [reply client/src/pages/Profile/index.js]: This isn't my code but i'll commit it anyway :P

File: client/src/pages/Profile/index.js [hunk 9]
```diff
@@ -1,183 +1,225 @@
...
-        )
-    }
+  constructor(props) {
+    super(props);
+    this.ResheduleInActive = this.ResheduleInActive.bind(this);
+    this.ResheduleActive = this.ResheduleActive.bind(this);
+    this.state = { showRegister: false };
+  }
+  ResheduleInActive() {
+    this.setState({ ...this.state, showRegister: false });
```
Comment (khattakdev) [inline client/src/pages/Profile/index.js]:
```suggestion
    this.setState({ showRegister: false });
```

**Review by khattakdev — CHANGES_REQUESTED** [review]

**Review by khattakdev — APPROVED** [review]
Looks good now

**Pull request merged**